rtslib/tcm.py

```diff
@@ -823,8 +826,8 @@ def _configure(self, config, size, wwn, hw_max_sectors):
         self._control("dev_size=%d" % size)
         if hw_max_sectors is not None:
             self._control("hw_max_sectors=%s" % hw_max_sectors)
-        self.enable = True
-
+        if enable is True:
+            self.enable = True
```

Comment:
Reviewers,

The dark green spacing means it is a tab right? If so, I repushed a patch with just spaces, but for some reason it does not seem to show here. If you clone my branch it looks correct, but in this web page it does not.

Reply:
Seems like spaces to me, so all good.